sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -147,6 +151,10 @@ public void sendEvent(EventBuilder eventBuilder) {
             eventBuilder.withTag(tagEntry.getKey(), tagEntry.getValue());
         }
 
+        // Note that extraTags are missing here because they only exist to be used by
+        // other systems (before this method is even called). For example: the logging
+        // integrations check the SLF4J MDC system for extras.
```

Comment:
Eh ya, it started as a note to self about why all the other fields were touched here and I think it turned out far more confusing than helpful.